libs/generator/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_library(block_generator STATIC generator.cpp)
```

Comment:
Not sure if it make sense to create library. May just directly use in the generator executable?

Reply:
It probably makes sense in terms of re-usability. If you place whole code into generator executable, you won't be able to call generation with its' API from somewhere else. So I think it just allows us to not be bounded with that. But that's only my thoughts on that, Misha maybe meant something different.

Reply:
Yes, the main reason is to reuse block generation on API level, mainly for testing